doc/source/user_guide/numpy_images.rst

```diff
@@ -2,78 +2,77 @@
 A crash course on NumPy for images
 ==================================
 
-Images manipulated by ``scikit-image`` are simply NumPy arrays. Hence, a
-large fraction of operations on images will just consist in using NumPy::
+Images in ``scikit-image`` are represented by NumPy ndarrays. Hence, many 
+common operations can be achieved using standard NumPy methods for 
+manipulating arrays:
 
     >>> from skimage import data
     >>> camera = data.camera()
     >>> type(camera)
     <type 'numpy.ndarray'>
 
-Retrieving the geometry of the image and the number of pixels: ::
+Retrieving the geometry of the image and the number of pixels:
```

Comment:
the ``::`` are needed by sphinx to display the code block below

Reply:
Hmm, I got confused by the GitHub UI editor. Looks like it parses the code only when there is a single `:`, and makes this without any reason. 🤔